index.html

```diff
@@ -1,16 +1,269 @@
+      <div align="center">
+        <span class="md-ellipsis">
+          Eclipse iceoryx™ is an inter-process-communication middleware that enables virtually limitless data transmissions at constant time
+        </span>
+      </div>
+    </div>
+  </nav>
+</header>
+
+    <div class="md-container" data-md-component="container">
+      <main class="md-main" data-md-component="main">
+        <div class="md-main__inner md-grid" width="100%">
+          <div class="md-content" data-md-component="content"  align="center">
+            <article class="md-content__inner md-typeset" style="max-width: 1000px">
+
+<p align="center">
+    <img src="https://user-images.githubusercontent.com/8661268/114321508-64a6b000-9b1b-11eb-95ef-b84c91387cff.png" width="50%">
+
+
+<table align="center">
+<thead>
+<tr>
+<th><h2>iceoryx</h2></th>
+<th><h2>iceoryx2</h2></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td width="50%">
+  <ul>
+    <li>C++</li>
+    <li>True zero-copy</li>
+    <li>Central daemon</li>
+    <li>Multiple messaging patterns
+      <ul>
+        <li>publish-subscribe</li>
+        <li>request-response</li>
+      </ul>
+    </li>
+    <li>Events</li>
+    <li>Wide range of platforms
+      <ul>
+        <li>Linux</li>
+        <li>Windows</li>
+        <li>macOS</li>
+        <li>FreeBSD</li>
+        <li>QNX</li>
+        <li>FreeRTOS</li>
+      </ul>
+    </li>
+    <li>Some language bindings
+      <ul>
+        <li>C</li>
+        <li>Rust (partial)</li>
+      </ul>
+    </li>
+    <li>Host2Host communication via DDS</li>
+  </ul>
+</td>
+<td>
+  <ul>
+    <li>Rust</li>
+    <li>True zero-copy</li>
+    <li>No central daemon</li>
+    <li>Multiple messaging patterns
+      <ul>
+        <li>publish-subscribe</li>
+        <li>request-response</li>
+        <li>pipeline (planned)</li>
+        <li>blackboard (planned)</li>
+      </ul>
+    </li>
+    <li>Events</li>
+    <li>Wide range of platforms
+      <ul>
+        <li>Linux</li>
+        <li>Windows</li>
+        <li>macOS</li>
+        <li>FreeBSD</li>
+        <li>Android (planned)</li>
+        <li>QNX (planned)</li>
+        <li>FreeRTOS (planned)</li>
+        <li>VxWorks (planned)</li>
+      </ul>
+    </li>
+    <li>Wide range of language bindings
+      <ul>
+        <li>C and C++ (planned)</li>
+        <li>Python (planned)</li>
+        <li>C# (planned)</li>
+        <li>and more</li>
+      </ul>
+    </li>
+    <li>Host2Host communication (planned)</li>
+    <li>Gateways to network protocols
+      <ul>
+        <li>DDS (planned)</li>
+        <li>gRPC (planned)</li>
+        <li>Zenoh (planned)</li>
+      </ul>
+    </li>
+    <li>Modular design allows custom extension, e.g. GPUs and Hypervisors</li>
+  </ul>
+</td>
+</tr>
+<tr>
+<td><a class="md-button" href="latest">Continue with iceoryx</a></td>
+<td><a class="md-button" href="https://github.com/eclipse-iceoryx/iceoryx2">Continue with iceoryx2</a></td>
```

Comment:
As a reader of the technical documentation, I would actually expect a link to https://docs.rs/iceoryx2/latest/iceoryx2/ here.

Why are you linking to GitHub here? That's slightly inconsistent with the other link.

Reply:
I didn't thought about that. Good point